src-v2/Not/Not.Blazor/TM/Forms/Components/NotForm.cs

```diff
@@ -29,6 +29,10 @@ protected void RegisterInjector<TInput>(string field, Func<MudBaseInput<TInput>>
     {
         ValidationInjectors.Add(field, MudValidationInjector.Create(mudInputInstanceGetter));
     }
+    protected void RegisterInjector<TInput>(string field, Func<MudPicker<TInput>> mudInputInstanceGetter)
```

Comment:
This overload is unnecessary, because there is a common ancestor to `MudBaseInput` and `MudPicker`. 

That said, it's acceptable solution, given the fact that this whole thing is a workaround.